editoast/src/views/test_app.rs

```diff
@@ -213,18 +224,45 @@ impl TestApp {
     pub fn get(&self, path: &str) -> TestRequest {
         self.server.get(&trim_path(path))
     }
+
     pub fn post(&self, path: &str) -> TestRequest {
         self.server.post(&trim_path(path))
     }
+
     pub fn put(&self, path: &str) -> TestRequest {
         self.server.put(&trim_path(path))
     }
+
     pub fn patch(&self, path: &str) -> TestRequest {
         self.server.patch(&trim_path(path))
     }
+
     pub fn delete(&self, path: &str) -> TestRequest {
         self.server.delete(&trim_path(path))
     }
+
+    pub async fn def_user(&self, user: &UserInfo, roles: HashSet<BuiltinRole>) {
```

Comment:
```suggestion
    pub fn def_user(&self, user: &UserInfo, roles: impl IntoIterator<Item = BuiltinRole>) {
```

Let's try to make that sync with `futures::executor::block_on`. Saves us an .await in tests.

Reply:
Done.